Directory.Build.props

```diff
@@ -9,5 +9,8 @@
       analyzers, which run during build. Refer to https://aka.ms/fxcopanalyzers to migrate to FxCop analyzers."
     -->
     <SuppressLegacyCodeAnalysisDeprecatedWarning>true</SuppressLegacyCodeAnalysisDeprecatedWarning>
+    <!-- Workaround continued use of netcoreapp2.1; suppress the NETSDK1138 warning. -->
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
Worth conditioning this on `TFM=netcoreapp2.1`, in case somebody checks in a `netcoreapp1.0` target or something?

Reply:
I think we'd notice :grin: